src/message.rs

```diff
@@ -0,0 +1,12 @@
+use super::schema::message;
+
+#[derive(Queryable)]
+#[derive(Insertable)]
+#[table_name="message"]
+pub struct Message {
+    pub id: i64,
+    pub content: String,
+    pub guild_id: i64,
+    pub author_id: i64,
+    pub channel_id: i64,
```

Comment:
Then make the internal API support `u64`. Have only the code that directly glues the database to the rest of the program use `i64.